gen-certs.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+set -e
+
+CACN=${CACN:-CaaSP Bootstrap CA}
```

Comment:
Since this outlives bootstrap, we should name it something else - e.g. CaaSP Internal CA